test/support/api.js

```diff
@@ -8,13 +8,13 @@ export const onDiscover = (url, links) =>
         .addLinks(links)
         .toObject())
 
-export const onGet = (url, path, resource) =>
-  nock(url)
+export const onGet = (url, path, resource, headers) =>
```

Comment:
Just adding headers as an arg probably won't be very extensible.

Can you wrap it in an object? e.g.

```
const onGet = (url, path, resource, { headers }) => ...
```